samples/camera/README.rst

```diff
@@ -0,0 +1,34 @@
+.. _camera:
```

Comment:
As written, this seems like a generic camera sample, but I think it's actually a very specific sample for an MT9M114 image sensor connected to a mimxrt1050_evk board.   The documentation should reflect this in the title, description, hardware requirements, and references.

Reply:
Not only for mt9m114 and mimxrt1050. The sample code uses generic API and no specific low level API used.
But so far, only mt9m114 driver and imxrt CSI driver support on Zephyr.

Reply:
Not only for mt9m114 and mimxrt1050. The sample code uses generic API and no specific low level API used.
But so far, only mt9m114 driver and imxrt CSI driver support on Zephyr.

Reply:
OK, thanks for the explanation.

Reply:
update the README of camera sample